x/leverage/keeper/internal_test.go

```diff
@@ -69,3 +69,42 @@ func (tk *TestKeeper) SetInterestScalar(ctx sdk.Context, denom string, scalar sd
 func (tk *TestKeeper) SetReserveAmount(ctx sdk.Context, coin sdk.Coin) error {
 	return tk.Keeper.setReserveAmount(ctx, coin)
 }
+
+// AllInvariants runs all invariants of the x/leverage module, including the inefficient ones.
+// This is only used in testing.
+func (tk *TestKeeper) AllInvariants() sdk.Invariant {
+	k := *tk.Keeper
```

Comment:
no need to copy I think.

Reply:
Simplified now in `(s *IntegrationTestSuite) checkInvariants(msg)` with no unnecessary duplication